frontend/src/app/testQueue/TestCard/TestCard.tsx

```diff
@@ -39,11 +41,16 @@ export const TestCard = ({
   const navigate = useNavigate();
   const timer = useTestTimer(
     testOrder.internalId,
-    testOrder.deviceType.testLength
+    testOrder.deviceType.testLength,
+    testOrder.timerStartedAt === null
+      ? undefined
+      : Number(testOrder.timerStartedAt)
```

Comment:
Explicitly passing in undefined when `timerStartedAt` is null because simply calling `Number(null)` evaluates to 0 which would then not correctly trigger the reset logic within the timer code